packages/fast-usdc/src/constants.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Status values for the StatusManager. Listed in order:
+ *
+ * - 'OBSERVED': TX is observed via EventFeed
+ * - 'ADVANCED': advance funds are available and IBC transfer is initiated (but
+ *   not necessarily settled)
+ * - 'SETTLED': settlement for matching advance received and funds dispersed
+ *
+ * @enum {(typeof TxStatus)[keyof typeof TxStatus]}
+ */
+export const TxStatus = /** @type {const} */ ({
+  /** when TX is observed via EventFeed */
+  OBSERVED: 'OBSERVED',
+  /** advance funds are available and IBC transfer is initiated */
+  ADVANCED: 'ADVANCED',
+  /** settlement for matching advance received and funds dispersed */
+  SETTLED: 'SETTLED',
```

Comment:
since it's in any enum it doesn't have to be ALL_CAPS.

please match the [TS enum style](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums)